test/model/src/include/model/driver/kv_workload.h

```diff
@@ -537,7 +537,8 @@ class kv_workload {
      * kv_workload::run_in_wiredtiger --
      *     Run the workload in WiredTiger.
      */
-    void run_in_wiredtiger(const char *home, const char *connection_config) const;
+    void run_in_wiredtiger(const char *home, const char *connection_config = nullptr,
```

Comment:
The function `verify_workload` calls `run_in_wiredtiger` without the last argument. While I was adding the new argument, I decided that I may as well make the other one optional. But I can take that out if need be.

Reply:
Up to you - I didn't see that existing usage since it wasn't in the diff. If there's a reason for the arguments to be optional then I'm fine to leave them as-is :-)